### PR TITLE
Rework RBAC env vars

### PR DESCRIPTION
**Pull request by dirkkul**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: test/acceptance/authz/rbac_auto_no_permissions_test.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ func TestAuthzAllEndpointsNoPermissionDynamically(t *testing.T) {
 	defer cancel()
 
 	compose, err := docker.New().WithWeaviate().WithRBAC().
-		WithRbacUser("admin-User", adminKey, "admin-User").
-		WithRbacUser("custom-user", customKey, "custom").
+		WithRbacAdmin("admin-User", adminKey).
+		WithRbacUser("custom-user", customKey).
```
Comment (moogacs) [inline test/acceptance/authz/rbac_auto_no_permissions_test.go:45]: **suggestion:** `withRbacUser()` has to be removed given that we will not support assigned non builtInRoles assignemnet

Comment (dirkkul) [reply test/acceptance/authz/rbac_auto_no_permissions_test.go:45]: The users still need to be added to the API key env vars

Comment (dirkkul) [reply test/acceptance/authz/rbac_auto_no_permissions_test.go]: reworked the user creation in tests and separeted them from RBAC

File: tools/dev/run_dev_server.sh [hunk 2]
```diff
@@ -70,7 +70,8 @@ case $CONFIG in
     AUTHORIZATION_ENABLE_RBAC=true \
     AUTHENTICATION_APIKEY_ALLOWED_KEYS='jane-secret-key,ian-secret-key,jp-secret-key' \
     AUTHENTICATION_APIKEY_USERS='jane@doe.com,ian-smith,jp-hwang' \
-    AUTHENTICATION_APIKEY_ROLES='viewer,editor,admin' \
+    AUTHORIZATION_ADMIN_USERS='admin' \
+    AUTHORIZATION_VIEWER_USERS='viewer' \
```
Comment (moogacs) [inline tools/dev/run_dev_server.sh:74]: **issue:** 
```suggestion
    AUTHORIZATION_ADMIN_USERS='jp-hwang' \
    AUTHORIZATION_VIEWER_USERS='ian-smith' \
```

File: tools/dev/run_dev_server.sh [hunk 3]
```diff
@@ -92,7 +93,8 @@ case $CONFIG in
...
```
Comment (moogacs) [inline tools/dev/run_dev_server.sh:97]: **issue:** 
```suggestion
    AUTHORIZATION_ADMIN_USERS='jp-hwang' \
    AUTHORIZATION_VIEWER_USERS='ian-smith' \
```

File: tools/dev/run_dev_server.sh [hunk 4]
```diff
@@ -118,7 +120,8 @@ case $CONFIG in
...
```
Comment (moogacs) [inline tools/dev/run_dev_server.sh:124]: **issue (blocking):** 
```suggestion
    AUTHORIZATION_ADMIN_USERS='jp-hwang' \
    AUTHORIZATION_VIEWER_USERS='ian-smith' \
```

File: tools/dev/run_dev_server.sh [hunk 5]
```diff
@@ -148,7 +151,8 @@ case $CONFIG in
...
```
Comment (moogacs) [inline tools/dev/run_dev_server.sh:155]: **issue (blocking):** 
```suggestion
    AUTHORIZATION_ADMIN_USERS='jp-hwang' \
    AUTHORIZATION_VIEWER_USERS='ian-smith' \
```

**Review by moogacs — APPROVED** [review]
LGTM, just small changes in the shell config

**Pull request merged**